<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Picture/DISPLAY.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVa">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVb">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIa">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIb">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIc">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;W&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberVa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberVb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberVc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberIa">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberIb">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberIc">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonstart">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/icon-start.png</normaloff>../Picture/icon-start.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonstop">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/icon-stop.png</normaloff>../Picture/icon-stop.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonmenu">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/icon-menu.png</normaloff>../Picture/icon-menu.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NumberkWh">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelkWh">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#00ff00;&quot;&gt;kWh&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonuptime">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/up-time.png</normaloff>../Picture/up-time.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttondowntime">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>230</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/down-time.png</normaloff>../Picture/down-time.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Picture/go-back.png</normaloff>../Picture/go-back.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
